viorng/VirtRNG Package/VirtRNG Package.vcxproj

```diff
@@ -104,6 +104,7 @@
     <OutDir>$(ProjectDir)..\Install\win8\x86\</OutDir>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Win10 Release|Win32'">
+    <DriverTargetPlatform>Desktop</DriverTargetPlatform>
```

Comment:
What will it take to move the platform to "Univeral". We would like to be able to build VirtRNG for the ARM64 platform eventually.

Reply:
In short: we should remove custom coinstaller from inf file.

Reply:
No other way than removing the coinstaller? I tried using an extension inf file without success and still didn't find a way to register the driver as a RNG provider without an installer.